solid-terms.ttl

```diff
@@ -195,3 +195,10 @@
     rdfs:label "where"@en ;
     rdfs:domain :Patch ;
     rdfs:range log:Formula .
+
+:storageQuota
+    a rdf:Property, owl:DatatypeProperty ;
```

Comment:
Domain and range?

Reply:
Domain is `solid:Account` and/or `pim:Storage`. Range could be `byte-unit:Unit`, or just `rdfs:Literal`. I'm not sure any would look for that. Should we have them in?

Reply:
Added `solid:Account` as domain for now, we can do range later once we find a home for #35 .